.github/workflows/main.yml

```diff
@@ -60,7 +60,7 @@ jobs:
         python3 tests/load_es_data.py tests/data/ne_110m_populated_places_simple.geojson geonameid
         python3 tests/load_es_data.py tests/cite/ogcapi-features/canada-hydat-daily-mean-02HC003.geojson IDENTIFIER
         python3 tests/load_mongo_data.py tests/data/ne_110m_populated_places_simple.geojson
-        #gunzip < tests/data/hotosm_bdi_waterways.sql.gz | psql -U postgres test
+        gunzip < tests/data/hotosm_bdi_waterways.sql.gz | psql postgresql://postgres:postgres@localhost:5432/test
```

Comment:
Can we use `${{ secrets.DatabasePassword }}` as the password?

Reply:
I switched because the workflow (and thus checks) failed when a secret wasn't set. But I think we could do something like `postgresql password: ${{ secrets.DatabasePassword || postgres }}`. The other part is that the test_postgresql_provider.py also needs access to the database password.
 [here](https://github.com/geopython/pygeoapi/blob/3de5271d0b768d4717ca64f6ae86d3db7da4ec0f/tests/test_postgresql_provider.py#L45-L48):  
```
 'data': {'host': '127.0.0.1',
          'dbname': 'test',
          'user': 'postgres',
          'password': 'postgres',
```
Would that have to be set as an environment variable in the workflow and then retrieved for the test?

Reply:
@tomkralidis  if `${{ secrets.DatabasePassword || postgres }}` works for you, I have a PR primed with it implemented.

Reply:
OK. Or can we wrap the update into this PR?